django_unicorn/static/unicorn/js/morphers/alpine.js

```diff
@@ -0,0 +1,35 @@
+export class AlpineMorpher {
+  constructor(options) {
+    // Check if window has Alpine and Alpine Morph
+    if (!window.Alpine || !window.Alpine.morph) {
+      throw Error(`
+Alpine morpher requires Alpine to be loaded. Add Alpine and Alpine Morph to your page.
+See https://www.django-unicorn.com/docs/custom-morphers/#alpine for more information.
+`);
+    }
+    this.options = options;
+  }
+
+  morph(dom, htmlElement) {
+    if (htmlElement) {
```

Comment:
@adamghill, frankly, I couldn't understand why morph is sometimes called with an empty string as "htmlElement".